src/main/java/com/onlinebanking/enums/ErrorMessage.java

```diff
@@ -1,18 +1,18 @@
-package com.onlinebanking.constant;
+package com.onlinebanking.enums;
 
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 
 /**
- * Class ErrorConstants will hold all the error messages used in the application.
+ * Class ErrorMessage will hold all the error messages used in the application.

```

Comment:
Comment should be updated to reflect this type as Enum and not class. 
Or
Class can be removed leaving it as "ErrorMessage will...."